src/bundle/ui-dev/src/modules/universal-discovery/hooks/useSelectedLocationsHelpers.js

```diff
@@ -0,0 +1,33 @@
+import { useCallback, useContext } from 'react';
+
+import { checkIsSelectable, checkIsSelected, checkIsSelectionBlocked } from '../helpers/selected.locations.helper';
+import {
+    AllowedContentTypesContext,
+    ContainersOnlyContext,
+    ContentTypesMapContext,
+    MultipleConfigContext,
+    SelectedLocationsContext,
+} from '../universal.discovery.module';
+
+export const useSelectedLocationsHelpers = () => {
```

Comment:
Note: I created a hook which wraps helpers so that instead of having to do something like this:
```js
import { checkIsSelectable, checkIsSelected, checkIsSelectionBlocked } from '../../helpers/locations.helper';
import {
    ContentTypesMapContext,
    SelectedLocationsContext,
    MultipleConfigContext,
    ContainersOnlyContext,
    AllowedContentTypesContext,
} from '../../universal.discovery.module';

const contentTypesMap = useContext(ContentTypesMapContext);
const [selectedLocations] = useContext(SelectedLocationsContext);
const [multipleItemsLimit] = useContext(MultipleConfigContext);
const containersOnly = useContext(ContainersOnlyContext);
const allowedContentTypes = useContext(AllowedContentTypesContext);
const isSelected = checkIsSelected({ location, selectedLocations });
const isNotSelectable = !checkIsSelectable({ location, contentTypesMap, allowedContentTypes, containersOnly });
const isSelectionBlocked = checkIsSelectionBlocked({ location, selectedLocations, multipleItemsLimit });
```
we can do:
```js
import { useSelectedLocationsHelpers } from '../../hooks/useSelectedLocationsHelpers';

const { checkIsSelectable, checkIsSelected, checkIsSelectionBlocked } = useSelectedLocationsHelpers();
const isSelected = checkIsSelected(location);
const isNotSelectable = !checkIsSelectable(location);
const isSelectionBlocked = checkIsSelectionBlocked(location);
```